tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -1590,6 +1634,209 @@ static struct bt_le_per_adv_sync_cb bap_pa_sync_cb = {
 	.synced = bap_pa_sync_synced_cb,
 };
 
+static void btp_send_pas_sync_req_ev(struct bt_conn *conn, uint8_t src_id,
+				     uint8_t advertiser_sid, uint32_t broadcast_id,
+				     bool past_avail, uint16_t pa_interval)
+{
+	struct btp_bap_pa_sync_req_ev ev;
+	struct bt_conn_info info;
+
+	(void)bt_conn_get_info(conn, &info);
+	bt_addr_le_copy(&ev.address, info.le.dst);
```

Comment:
We could consider implementing a helper function to copy address from `bt_conn_info` to an address pointer to make this a single line. 

Alternative, there is also the `bt_conn_get_dst` function that can be used to get the address directly, so this could be

```suggestion
	bt_addr_le_copy(&ev.address, bt_conn_get_dst(conn));
```

Reply:
This applies for all cases where we want to copy the address of a `conn` to an event